redirects.yml

```diff
@@ -33,7 +33,8 @@ redirects:
   'solutions/observability/apps/upstream-opentelemetry-collectors-language-sdks.md': 'solutions/observability/apm/upstream-opentelemetry-collectors-language-sdks.md'
   'solutions/observability/apps/collect-metrics.md': 'solutions/observability/apm/collect-metrics.md'
   'solutions/observability/apps/limitations.md': 'solutions/observability/apm/limitations.md'
-  'solutions/observability/apps/resource-atrributes.md': 'solutions/observability/apm/resource-attributes.md'
+  'solutions/observability/apps/resource-atrributes.md': 'solutions/observability/apm/attributes.md'
+  'solutions/observability/apps/resource-attributes.md': 'solutions/observability/apm/attributes.md'
```

Comment:
duplicated lines, is this expected?

Reply:
There's a typo in the first, probably due to a bad link in origin.

Reply:
brilliant!